src/pytkdocs/objects.py

```diff
@@ -173,20 +173,26 @@ def relative_file_path(self) -> str:
 
         If the relative file path cannot be determined, the value returned is `""` (empty string).
         """
-        top_package_name = self.path.split(".", 1)[0]
-        try:
-            top_package = sys.modules[top_package_name]
-        except KeyError:
+
+        subspace_count = len(self.path.split("."))
+        namespaces = [".".join(self.path.split(".", maxsplit=maxsplit)[:-1]) for maxsplit in range(1, subspace_count)]
+
+        for namespace in namespaces:
             try:
-                importlib.import_module(top_package_name)
-            except ImportError:
+                top_package = sys.modules[namespace]
+            except KeyError:
+                try:
+                    importlib.import_module(namespace)
+                except ImportError:
+                    return ""
```

Comment:
We return early because if we couldn't import that namespace, we are sure that we could not import the subspace neither, right?